docs/configuration/configuration.md

```diff
@@ -106,6 +106,10 @@ global:
   # change in the future.
   [ target_limit: <int> | default = 0 ]
 
+  # Per-job limit on the number of targets dropped by relabeling
```

Comment:
This is not per job, but per scrape pool, which to the user is exposed as a scrape config (there can be multiple jobs per scrape config if targets have a `job` label set that overrides the scrape config's default job name). So I'd say "Per-scrape-config limit ..." or "Limit per scrape config ..." (less hyphenated) or something like that.